.github/workflows/02-pr-check.yml

```diff
@@ -0,0 +1,21 @@
+name: Check PR Branches
+on:
+  pull_request:
+    types:
+      - opened
+      - edited
+      - reopened
+      - synchronize
+jobs:
+  check:
+    name: Only allow develop into main
```

Comment:
Are we likely to ever want to do hotfixes into main?

Reply:
Hmmm, I suppose that is a possibility. Though, technically speaking, the status check itself doesn't stop you from merging into main, it just throws up a red flag to catch cases where we accidentally open a PR against `main` instead of `develop`, and we'd need to set additional branch protection rules on a per-repo basis to actually make this a required status check, and we could override it if we did need to merge something to main.

But yeah, I am feeling a little mixed on how useful this status check actually is, tbh. On one hand, accidentally merging into `main` is more likely to happen if we're merging `develop` more often, since they will be fewer commits different from any feature branch, and it can lead to some headaches if we end up with conflicts. But it also feels weird to have a status check that's not totally required...

What would you think about keeping it in, evaluating how useful it actually is for a month or two, then reassessing?